package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "build": "node ./scripts/verify-packages-versions.js && lerna run build:types,build:transpile,build:bundle",
+    "build": "node ./scripts/verify-packages-versions.js && run-s build:types build:transpile build:bundle",
```

Comment:
cc @JamesHenry I reverted this here, as it seems that we sometimes ended up with an inconsistent state because of this (??) TBH we are not quite sure what is going on, but in multiple tests in weird ways, e.g. https://github.com/getsentry/sentry-javascript/actions/runs/4132316633/jobs/7140902414

I am not _really_ sure that this is the actual problem, but it's my best guess so far that some internal race condition or whatever leads to something being overwritten or not properly build... Will investigate some more!

Reply:
Sorry about that @mydea - I'll take a look.

FYI I invited you on Slack Connect, sent to your sentry email, so you can also reach me there

Reply:
@mydea At a quick glance it looks like tests depend on builds having run already, at the very least `build:types`?

We can naturally extend our existing task pipelines to let the lerna task runner handle coordinating this for us. I'll put up a PR

Reply:
There are also a couple of missing relationships from the package graph right (because the relevant package.json files are missing them)

Reply:
No worries, again I am not even really sure this was the problem, we seem to have an issue with a bunch of CI caching stuff coming together. Thanks for helping us figure this out!

I've put a request with our IT to get a shared slack channel, it is pending - looking forward to it going through!

I was also thinking about getting rid of our CI build cache and just leverage Nx cache. It would probably streamline stuff quite a bit!

Reply:
Cacheable tests PR which should hopefully resolve the issues you've been seeing as well: https://github.com/getsentry/sentry-javascript/pull/7108